floodviz/static/js/map.js

```diff
@@ -61,10 +63,10 @@
 				.attr("transform", function (d) {
 					return "translate(" + projection(d.geometry.coordinates) + ")";
 				})
-				.attr("id", function(d) {
-					if (classname === 'gage-point') {
-						return 'map' + d.properties.id;
-					} else { return '';}
+				.attr("id", function (d) {
+					if (property_for_id && d.properties[property_for_id]) {
+						return 'map' + d.properties[property_for_id];
+					}
```

Comment:
I added an else clause, but I don't want to use a ternary operator because I find them hard to read.